src/nodedef.cpp

```diff
@@ -682,6 +682,8 @@ void ContentFeatures::updateTextures(ITextureSource *tsrc, IShaderSource *shdsrc
 	switch (drawtype) {
 	default:
 	case NDT_NORMAL:
+		material_type = (alpha == 255) ?
```

Comment:
The default for alpha is 255, but i'm not sure any mod nodes other than liquids use the 'alpha' nodedef, they seem to use 'use texture alpha' instead. Also, semi-translucency is buggy in MT so not really usable.
So perhaps  this check is not needed?